src/nvim/path.c

```diff
@@ -329,20 +329,25 @@ int vim_fnamencmp(char_u *x, char_u *y, size_t len)
 #endif
 }
 
-/*
- * Concatenate file names fname1 and fname2 into allocated memory.
- * Only add a '/' or '\\' when 'sep' is TRUE and it is necessary.
- */
-char_u *concat_fnames(char_u *fname1, char_u *fname2, int sep)
-  FUNC_ATTR_NONNULL_RET
+/// Concatenate file names fname1 and fname2 into allocated memory.
+/// 
+/// Only add a '/' or '\\' when 'sep' is TRUE and it is necessary.
```

Comment:
doh, I meant `true`, since it's a `bool` and not just an `int` assigned to `(TRUE|FALSE)`.